requirements.txt

```
@@ -6,3 +6,4 @@ autograd
 toml
 appdirs
 semantic_version==2.6
+dask[delayed]==2.9.2
```

Looks like we need to revert this for the TensorFlow 1.X tests to pass :(
```suggestion
dask[delayed]
```